common/lib/xmodule/xmodule/js/spec/video/html5_video_spec.js

```diff
@@ -220,7 +220,7 @@
                         }).done(done);
                     });
 
-                    it('set new inccorrect values', function() {
+                    it('set new incorrect values', function() {
```

Comment:
This test seems to be flaky.  I saw it fail once, and couldn't bear to let the typo survive.  I'm not sure what the process is for external contributors marking tests flaky, so I haven't created any tickets about it.

Reply:
If you know it to be flaky, I don't see any issue with you following the updated flaky test process.